### DISCUSSION TITLE
Suggestion for a 3D/2D mode transition

### DISCUSSION THREAD

**Discussion by DennoTheMenno** [osmandapp/OsmAnd#18485] [Ideas]
I'm thinking it would be a nice feature for an "AUTO" mode, where 3D mode slowly transitions to 2D mode when slowing to a stop.
This would allow a better viewing of the surrounding area when stopped (top down view)

Just a suggestion.

Thanks.

**Comment by DmitryAlexei** [comment]
Good suggestion, thanks!
